pkgs/development/haskell-modules/configuration-common.nix

```
@@ -1330,6 +1330,10 @@ self: super: {
   # https://github.com/ennocramer/monad-dijkstra/issues/4
   monad-dijkstra = dontCheck (doJailbreak super.monad-dijkstra);
 
```

```suggestion

  # Fixed upstream but not released to Hackage yet:
  # https://github.com/k0001/hs-libsodium/issues/2
```

We generally try to keep comments like this in the nixpkgs code so that it is easy for the nixpkgs maintainers to know when overrides can be removed.